src/ServicePulse.Host/vue/public/mockServiceWorker.js

```diff
@@ -0,0 +1,287 @@
+/* eslint-disable */
+/* tslint:disable */
+
+/**
+ * Mock Service Worker (2.2.2).
+ * @see https://github.com/mswjs/msw
+ * - Please do NOT modify this file.
+ * - Please do NOT serve this file on production.
```

Comment:
Does this need to be explicitly excluded from application packaging?

Reply:
yes